src/tile-component.coffee

```diff
@@ -0,0 +1,364 @@
+_ = require 'underscore-plus'
+{toArray} = require 'underscore-plus'
+{$$} = require 'space-pen'
+
+TokenIterator = require './token-iterator'
+DummyLineNode = $$(-> @div className: 'line', style: 'position: absolute; visibility: hidden;', => @span 'x')[0]
+AcceptFilter = {acceptNode: -> NodeFilter.FILTER_ACCEPT}
+WrapperDiv = document.createElement('div')
+TokenTextEscapeRegex = /[&"'<>]/g
+MaxTokenLength = 20000
+
+cloneObject = (object) ->
+  clone = {}
+  clone[key] = value for key, value of object
+  clone
+
+module.exports =
+class TileComponent
```

Comment:
This diff is the result of an unfortunate false positive: ideally, git should have recognized `LinesComponent` as a whole new file, and `TileComponent` as a rename from the previous `lines-component.coffee`.